### PR TITLE
Moving to steering

### PR DESCRIPTION
**Pull request by apeltzer**
This moves me to steering and out of core, will take care of the removal of my accounts from slack and github now too.

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1846"></a>

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
Also need to duplciate on a PR against `main` (ping @mashehu )

File: markdown/governance.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ The steering committee will meet regularly to discuss the project, funding and p
 <a class="btn btn-light rounded-pill" href="https://github.com/ggabernet"><i aria-hidden="true"></i>Gisela Gabernet</a>
 <a class="btn btn-light rounded-pill" href="https://github.com/drpatelh"><i aria-hidden="true"></i>Harshil Patel</a>
 <a class="btn btn-light rounded-pill" href="https://github.com/ewels"><i aria-hidden="true"></i>Phil Ewels</a>
+<a class="btn btn-light rounded-pill" href="https://github.com/apeltzer"><i aria-hidden="true"></i>Alexander Peltzer</a>
+
```
Comment (maxulysse) [inline markdown/governance.md:43]:
```suggestion
```

**Review by maxulysse — APPROVED** [review]
Sadly I approve this PR

**Comment by mashehu** [comment]
Will do this in this PR: #1845

**Pull request closed without merging**